examples/step-27/step-27.cc

```diff
@@ -228,7 +228,7 @@ namespace Step27
     // finite element, namely that is obtained by iterating a
     // 2-point Gauss formula as many times as the maximal exponent we use for
     // the term $e^{i{\bf k}\cdot{\bf x}}$:
-    QGauss<1>      base_quadrature(2);
+    QGauss<1>      base_quadrature(2); //Has to be modified for hp-elements to make it depending on the degree of the finite element?
```

Comment:
As explained in the comments above this line, I think this is ok. This quadrature formula is iterated as many times as necessary to deal with the highest polynomial degree. Just remove the comment again.